tools/klee-replay/klee-replay.c

```diff
@@ -192,12 +192,17 @@ static void run_monitored(char *executable, int argc, char **argv) {
 #endif
 
     if (!rootdir) {
+      if (chdir(replay_dir) != 0) {
```

Comment:
This is broken now, because executable can no longer be a relative path to the current dir. For example

`klee-replay b2sum path-to-ktest-file` no longer works because `execv` can't find b2sum in the `replay_dir`